src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -324,6 +325,14 @@ public static int getPushInfo(String toUser) {
         return 0;
     }
 
+    public static void updatePushDevices(OtpUser otpUser) {
```

Comment:
Is there any expectation to add Javadoc in this project?

Reply:
I added the JavaDoc in https://github.com/ibi-group/otp-middleware/pull/193/commits/c7b05affa1b28e4efc68f502f4feded6fc9596aa.